fv3config/_asset_list.py

```diff
@@ -102,71 +116,45 @@ def get_asset_dict(source_location, source_name, target_location='',
-    recurse to subdirectories within source_directory.
-
-    Args:
-        source_directory (str): path to local directory from which to generate
-            asset_list.
-        target_location (str, optional): target_location used for generated assets.
-            Defaults to '' which is root of run-directory.
-        copy_method (str, optional): copy_method used for generated assets. Defaults
-            to 'copy'.
-
-    Returns:
-        list: a list of asset dictionaries
-    """
-    asset_list = []
-    for root, dirs, files in os.walk(source_directory):
-        if root == source_directory:
-            file_target_location = target_location
-        else:
-            file_target_location = os.path.join(target_location,
-                                                os.path.basename(root))
-        for file in files:
-            asset_list.append(get_asset_dict(os.path.join(source_directory, root),
-                                             file,
-                                             target_location=file_target_location,
-                                             copy_method=copy_method))
-    return asset_list
+def _get_gcloud_fs():
+    return gcsfs.GCSFileSystem(project=_get_gcloud_project())
 
 
-def asset_list_from_gs_bucket(url, target_location=''):
-    """Return asset_list from all files that begin with a google cloud storage url.
-    Will not recurse to files that are in "sub-directories" of the url.
+def asset_list_from_path(location, target_location='', copy_method='copy'):
+    """Return asset_list from all files within a given path.
 
     Args:
-        url (str): google cloud storage url from which to generate asset_list.
+        location (str): local path or google cloud storage url.
         target_location (str, optional): target_location used for generated assets.
             Defaults to '' which is root of run-directory.
+        copy_method ('copy' or 'link', optional): whether to copy or link assets,
+            defaults to 'copy'. If location is a google cloud storage url, this option
+            is ignored and files are copied.
 
     Returns:
         list: a list of asset dictionaries
         """
+    if is_gsbucket_url(location):
+        copy_method = 'copy'
+        fs = _get_gcloud_fs()
+    else:
+        fs = LOCAL_FS
     asset_list = []
-    # TODO: use gcsfs instead of gsutil for following code
-    stdout_str = Popen(['gsutil', 'ls', url], stdout=PIPE).stdout.read()
-    path_list = stdout_str.decode().split('\n')[:-1]
-    for path in path_list:
-        dirname, basename = os.path.split(path)
-        if basename != '':
-            asset_list.append(get_asset_dict(dirname,
-                                             basename,
-                                             target_location=target_location))
+    path_list = fs.walk(location)
+    for dirname, _, files in path_list:
+        print(dirname, files, target_location)
```

Comment:
perhaps removing this print statement, which i assume was there for debugging purposes.